config/eloquent-driver.php

```diff
@@ -2,8 +2,61 @@
 
 return [
 
+    'connection' => env('STATAMIC_ELOQUENT_CONNECTION', ''),
+    'table_prefix' => env('STATAMIC_ELOQUENT_PREFIX', ''),
+
+    'assets' => [
+        'driver' => 'eloquent',
+        'container-model' => \Statamic\Eloquent\Assets\AssetContainerModel::class,
+        'model' => \Statamic\Eloquent\Assets\AssetModel::class,
+    ],
+
+    'blueprints' => [
+        'driver' => 'eloquent',
+        'blueprint-model' => \Statamic\Eloquent\Fields\BlueprintModel::class,
```

Comment:
Should config keys be `snake_case`? I believe snake case is used everywhere else in laravel/statamic configs.

Reply:
Yeah they should, changed.